### DISCUSSION TITLE
could not find the package "Wasm.Common"

### DISCUSSION THREAD

**Discussion by angelosun** [apache/skywalking#9019] [Q&A] [answered]
I tried to build the skywalking source code, however I was to run into a error as below:
class  org.apache.skywalking.oap.server.receiver.envoy.als.mx.ServiceMetaInfoAdapter could not find the package "Wasm.Common"
Dose anyone have any suggestion?  thanks in advance!!

**Comment by kezhenxu94** [comment, marked as answer]
Read the doc, they are generated at build time
